### ISSUE TITLE
Floating point exception (core dumped)

### ISSUE THREAD

**Issue by davetapley** [duckdb/postgres_scanner#69]
### What happens?

```
Floating point exception (core dumped)
```

### To Reproduce

In Postgres:
```sql
create table fail(n numeric(12,7));
insert into fail values (32.8875000);
```

In DuckDB CLI:
```sql
INSTALL postgres;
LOAD postgres;
CALL postgres_attach('...')
select * from fail;
```

### OS:

Ubuntu 20.04.5 LTS

### PostgreSQL Version:

15.1

### DuckDB Version:

v0.6.1-dev83 dfae126ebb

### DuckDB Client:

CLI

### Full Name:

Dave Tapley

### Affiliation:

JE Fuller

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by hannes** [comment]
I get a wrong result on this one

**Comment by davetapley** [comment]
@hannes same OS / version? 🤔

**Comment by Mytherin** [comment]
Thanks for the report! This should be fixed now in #111.
